src/test/resources/com/puppycrawl/tools/checkstyle/filters/suppressionpatchxpathfilter/OverloadMethodsDeclarationOrder/context/Test.java

```diff
@@ -0,0 +1,9 @@
+package TreeWalker.coding.OverloadMethodsDeclarationOrder;
+
+public class Test {
+    public void foo(int i) {} // OK
+    public void foo(String s) {} // OK
+    public void notFoo() {}  // violation without filter
```

Comment:
Are you sure this comment is on the right place? The violation in the expected.txt shows line 7. Same for below cases.

Reply:
sorry, I have corrected it.